Nav.php

```diff
@@ -202,6 +202,32 @@ public function renderItem($item)
     }
 
     /**
+     * Check if parent item is visible
+     * @param array $parentItem
+     * @return boolean
+     */
+    protected function isParentVisible($parentItem)
```

Comment:
Name `isParentVisible()` is not appropriate.
Should be `isItemVisible()`, to be consistent with `isItemActive()`.